### PR TITLE
Fixed translation in explore-intro

### PR DESCRIPTION
**Pull request by FL3SH**
Hi,
I fixed small omission in explore intro.

### CODE REVIEW

**Comment by FL3SH** [comment]
I signed it

**Comment by FL3SH** [comment]
I signed it

File: content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 1]
```diff
@@ -51,7 +51,7 @@ <h3>Podsumowanie:</h3>
                 </div>
                 <div class="content__box content__box_fill">
                         <p><i>
-                            Pod to grupa jednego lub wielu kontenerów aplikacji (takich, jak Docker czy rkt) zawierający współdzielony storage (volumes), adres IP i informacje, jak mają być uruchamiane.
+                            Pod to grupa jednego lub wielu kontenerów aplikacji (takich, jak Docker czy rkt) zawierająch współdzieloną przestrzń dyskową (volumes), adres IP i informacje, jak mają być uruchamiane.
```
Comment (mfilocha) [inline content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html:54]:
```suggestion
                            Pod to grupa jednego lub wielu kontenerów aplikacji (takich, jak Docker czy rkt) zawierających współdzieloną przestrzeń dyskową (volumes), adres IP i informacje, jak mają być uruchamiane.
```

Comment (mfilocha) [reply content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html:54]: Thanks for your contribution! Please create a new PR with these two typos.

Comment (FL3SH) [reply content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html:54]: Sorry, for that, I was struggling with this PR and I have to fork repo a few times and I finally did typos...

Comment (mfilocha) [reply content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html:54]: No worries!

**Pull request merged**